policies/pb/policies.proto

```diff
@@ -49,5 +50,10 @@ message DatasetRes {
   string id = 1;
   string agent_group_id = 2;
   string policy_id = 3;
-  repeated string sink_ids = 4;
+  bool valid = 4;
```

Comment:
now that we will have our first users and production environments, we need to start thinking about backwards compatibility here - I'm pretty sure you should never change a field's number, you can only add new ones. we should read up on this

https://earthly.dev/blog/backward-and-forward-compatibility/
https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers